vizro-core/docs/pages/development/documentation-style-guide.md

```diff
@@ -0,0 +1,143 @@
+# Vizro documentation style guide
+
+This is the style guide we apply to the [Vizro documentation](https://vizro.readthedocs.io/en/stable/).
+
+We ask anyone kind enough to contribute documentation changes to follow this style for consistency and simplicity.
+
+What follows is a set of lightweight guidelines rather than rules. There are always edge cases and exceptions, and if it's not obvious what the style should be, consult the [Microsoft style guide](https://docs.microsoft.com/en-gb/style-guide/welcome/) for an example of good practice. We also use the [INCITS Inclusive Terminology Guidelines](https://standards.incits.org/apps/group_public/download.php/131246/eb-2021-00288-001-INCITS-Inclusive-Terminology-Guidelines.pdf).
+
+## Vizro lexicon
+
+The names of our products are **Vizro** and **Vizro-AI**.
+
+We refer to other products using their preferred capitalization. For example, Dash and Plotly are always capitalized, except where given as Python package names `dash` and `plotly`.
+
+Vizro components are named using lower case:
+
+> "Here is a guide to using containers..."
+
+Use code font when referring to the component as a class or object:
+
+> "To add a `Container` to your page..."
+
```

Comment:
Fussy point but something I find myself editing quite a bit. What happens if you pluralise these? Is it `Containers` or `Container`s? I go for the latter but we're not consistent, and maybe it's too fiddly to enforce.

Reply:
It's annoying isn't it -- I don't like to be pernickety about it either. If possible, without overt textual gymnastics, I'd write around it. So use the phrase `Container` objects rather than `Container`s or something, but only if it make sense to do it and doesn't make the text more cluttered.